.github/workflows/binaries.yml

```diff
@@ -2,17 +2,18 @@ name: pypi
 on:
   workflow_call:
 jobs:
-  runs-on: ubuntu-latest
-  steps:
-    - name: Checkout 🛎️Binaries
-      uses: actions/checkout@v2
-      with:
-        repository: unifyai/binaries
-        path: binaries
-        persist-credentials: false
+  release_binaries:
```

Comment:
I think we should instead add this name aside the `name` field on line [1](https://github.com/unifyai/ivy/pull/26867/files#diff-bdfe89ee8491e51e3f81ada685c51d24188416d8b529b443ee6085a0e0fd4a4aR1). I think given that this workflow's name is the same as that of `pypi.yml`, we have this problem where the `binaries` workflow doesn't get named correctly. 
And yeah the `release-binaries` title sounds good to me
Thanks @Sai-Suraj-27 😄

Reply:
@vedpatwardhan sure, I added it as the name on line-1, but I think it should be added under `jobs` also? I'm not sure but in all the other `.yml` files there is a name for the job under `jobs:` like `run_tests:`, `fix-linting:`, `build:`, `publish-release-docs:` , etc. So, I guess we should add the `release-binaries` name both on line 1, and immediately under the `jobs:` on line 4.

Reply:
yep sounds good! Thanks @Sai-Suraj-27 😄